pkg/artifactory/resource_artifactory_permission_target_test.go

```diff
@@ -196,7 +192,7 @@ func TestAccPermissionTarget_full(test *testing.T) {
 
 func TestAccPermissionTarget_addBuild(t *testing.T) {
 	_, permFqrn, permName := mkNames("test-perm", "artifactory_permission_target")
-	//_, _, repoName := mkNames("test-perm-repo", "artifactory_local_repository")
+	//_, _, repoName := mkNames("test-perm-repo", "artifactory_local_generic_repository")
```

Comment:
I can't imagine why not delete these now. We've been asked to hold off on this a bit. So, why not clean up?

Reply:
removed